sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs

```diff
@@ -2388,6 +2694,25 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct VirtualMachineProvisioningState : System.IEquatable<Azure.ResourceManager.Avs.Models.VirtualMachineProvisioningState>
```

Comment:
this is also very likely to aollide with types in other RPs such as compute where VirtualMachine comes from.
Could we add some prefix to specify this in more details?